.github/workflows/ci.yaml

```diff
@@ -91,10 +91,10 @@ jobs:
       - uses: actions/checkout@v3
       - uses: mobilecoinfoundation/actions/sgxsdk@main
         with:
-          version: 2.18.100.3
+          version: 2.19.100.3
```

Comment:
Note: That there are some comments in the code referencing the 2.18, for example <https://download.01.org/intel-sgx/sgx-linux/2.18/docs/Intel_SGX_Developer_Reference_Linux_2.18_Open_Source.pdf#%5B%7B%22num%22%3A303%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C94.5%2C341.25%2C0%5D>
I didn't update these as they feel like it will be a constant maintenance burden when the implementations haven't changed. 

But I'm not strongly against updating them.